src/CheckersBoard.java

```diff
@@ -111,22 +112,22 @@ public int findCoordinates(String s){
 	 */
 	public boolean correctOwner(int i){
 		if(turn == 'x'){	//if its x's turn, makes sure the spot is owned by 'x'
-			if(pieces.get(i) == 'x' || pieces.get(i) == 'X'){
-				return true;
-			}else{
-				return false;
-			}
+		if(pieces.get(i) == 'x' || pieces.get(i) == 'X'){
+			return true;
+		}else{
+			return false;
 		}
+	}
 		if(turn == 'o'){	//if its o's turn, makes sure the spot is owned by 'o'
-			if(pieces.get(i) == 'o' || pieces.get(i) == 'O'){
-				return true;
-			}else{
-				return false;
-			}
+		if(pieces.get(i) == 'o' || pieces.get(i) == 'O'){
+			return true;
+		}else{
+			return false;
 		}
-		return false;
 	}
-	
+	return false;
```

Comment:
And I'll give you a bonus 50 XP if you rewrite this function as a one-liner!  [Hint.](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html)